### PR TITLE
Add pinned SVG icon

### PR DESCRIPTION
**Pull request by abeddow91**
## What is the purpose of this change?
There is a desire to import the pinned icon from Source as it is used in both dotcom rendering and apps rendering in several places in the liveblog. See here for its inclusion in the [Source icon figma file](https://www.figma.com/file/Ai7AELHC6KCz38qKZkvuHo/%E2%97%90-Icons?node-id=55%3A2). 

As the original SVG was designed to be 24x24, we are using a hack to make the viewbox 30x30 (thanks @mxdvl ) in keeping with the other icons in Source

## What does this change?
- Adds the pinned icon SVG to source-react-components
- Adds the icon to storybook
- Updates the changeset

## Screenshots

Screenshot 2022-03-03 at 16 04 59

### Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [x] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### Cross browser and device testing

-   [ ] Tested with touch screen device

### Responsiveness

-   [ ] Tested at all breakpoints
-   [ ] Tested with with long text
-   [ ] Stretched to fill a wide container

### Documentation

-   [ ] Full API surface area is documented in the README
-   [x] Examples in Storybook

### CODE REVIEW

File: packages/@guardian/source-react-components/src/icons/SvgPinned.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { iconSize } from '@guardian/source-foundations';
+import type { IconProps } from './types';
+
+export const SvgPinned = ({ size }: IconProps): EmotionJSX.Element => (
+	<svg
+		viewBox="0 0 24 24"
```
Comment (SiAdcock) [inline packages/@guardian/source-react-components/src/icons/SvgPinned.tsx:7]: Can we apply the @mxdvl hack to make the viewbox 30x30, in keeping with the other icons in Source?

```suggestion
		viewBox="-3 -3 30 30"
```

Comment (abeddow91) [reply packages/@guardian/source-react-components/src/icons/SvgPinned.tsx]: Absolutely! All updated and commits squashed :)

Comment (mxdvl) [reply packages/@guardian/source-react-components/src/icons/SvgPinned.tsx]: I'll have a look at moving all the icons to a new format on 10% day

**Review by SiAdcock — DISMISSED** [review]
Awesome! Thanks 🙏

**Review by SiAdcock — APPROVED** [review]
Awesome, thanks @abeddow91 💎

**Pull request merged**